LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -126,7 +126,13 @@ public async Task<bool> SendEventAsync(LoRaDeviceTelemetry telemetry, Dictionary
                     // Enable retry for this send message, off by default
                     this.SetRetry(true);
 
-                    var messageJson = JsonConvert.SerializeObject(telemetry, Formatting.None);
+                    var settings = new JsonSerializerSettings
```

Comment:
We could have the settings as an static and reuse every time, instead of creating a new one for every serialization?

Reply:
Out of curiosity: did you make this change to avoid sending `"dupmsg": false`?
If that is the case, what about using the NullValueHandling?

Reply:
Since the DupMsg is a nullable, it would actually emit "dumpmsg": null which is not what I want. I can make sure that we use a static for the settings, makes sense. Thanks.